extensions/amp-subscriptions/0.1/local-subscription-platform.js

```diff
@@ -182,12 +182,18 @@ export class LocalSubscriptionPlatform {
   getEntitlements() {
     return this.urlBuilder_.buildUrl(this.authorizationUrl_,
         /* useAuthData */ false)
-        .then(fetchUrl =>
+        .then(fetchUrl => {
+          if (this.serviceAdapter_.getEncryptedDocumentKey()) {
+            //todo(chenshay): if crypt, switch to 'post'
```

Comment:
nit, but we have bunch of tools in AMP that require it be uppercase `TODO`

Reply:
done